.github/workflows/webviz-subsurface-components.yml

```diff
@@ -155,7 +181,7 @@ jobs:
         run: npm run build-storybook --prefix ./react
 
       - name: 📚 Update Storybook and GitHub demo application
-        if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || contains(github.event.head_commit.message, '[deploy test]')) && matrix.python-version == '3.8'
+        if: github.event_name == 'push' && (github.ref == 'refs/heads/master' || contains(github.event.head_commit.message, '[deploy test]')) && matrix.python-version == '3.8' && failure() && matrix.e2e-tests
```

Comment:
For my understanding: Why do we want to update official storybook only on failure?

Reply:
I think I did some kind of mistake while I was doing some "search and replace" stuff. Its a complete mistake, I will undo it.